src/java/com/zimbra/ssdb/SSDBEphemeralStore.java

```diff
@@ -47,14 +47,28 @@
 public class SSDBEphemeralStore extends EphemeralStore {
     public static String SSDB_EPHEMERAL_STORE = "ssdb";
     private JedisPool pool;
-    public SSDBEphemeralStore(JedisPool pool) {
-        this.pool = pool;
+    private String url;
+    public SSDBEphemeralStore(String url) throws ServiceException {
+        this.url = url;
+        pool = getPool(url);
         setAttributeEncoder(new SSDBAttributeEncoder());
     }
+    private Jedis getResource() throws ServiceException {
+        Jedis jedis;
+        try{
+            jedis = pool.getResource();
+            jedis.ping();
```

Comment:
Perhaps add a comment like "Make sure the pool is functioning correctly - some problems will cause an exception to be thrown here"